src/util.cpp

```diff
@@ -227,211 +182,210 @@ void split_LU(Eigen::MatrixXd* A, Eigen::MatrixXd* L, Eigen::MatrixXd* U) {
 }
 
 double rcond_1_getf(Eigen::MatrixXd* A_LU, double A_1_norm) {
-    int n = A_LU->rows();
+    int64_t n = A_LU->rows();
     assert(A_LU->cols() == n);
     double rcond = 10.0;
-    int info = LAPACKE_dgecon(LAPACK_COL_MAJOR, '1', n, A_LU->data(), n, A_1_norm, &rcond);
+    int64_t info = lapack::gecon(Norm::One, n, A_LU->data(), n, A_1_norm, &rcond);
     assert(info == 0);
     return rcond;
 }
 
 double rcond_1_potf(Eigen::MatrixXd* A_LLT, double A_1_norm) {
-    int n = A_LLT->rows();
+    int64_t n = A_LLT->rows();
     assert(A_LLT->cols() == n);
     double rcond = 10.0;
-    int info = LAPACKE_dpocon(LAPACK_COL_MAJOR, 'L', n, A_LLT->data(), n, A_1_norm, &rcond);
+    int64_t info = lapack::pocon(Uplo::Lower, n, A_LLT->data(), n, A_1_norm, &rcond);
```

Comment:
`lapack::pocon` assumes col-major?

Reply:
All lapackpp functions assume column-major. blaspp lets you choose row-major or column-major, though.